src/SprykerEco/Zed/NewRelic/Business/Model/RecordDeployment.php

```diff
@@ -13,8 +13,8 @@
 
 class RecordDeployment implements RecordDeploymentInterface
 {
-    const STATUS_CODE_SUCCESS = 200;
-    const STATUS_CODE_REDIRECTION = 300;
+    protected const STATUS_CODE_SUCCESS = 200;
+    protected const STATUS_CODE_REDIRECTION = 300;
```

Comment:
Should it be public for BC reasons?

Reply:
IMO not as this is for internal use only and the code is not used by any project ATM as it was not working.